libs/logs.d.ts

```diff
@@ -1,10 +1,64 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-export default log;
+import {
+  APIGatewayProxyEvent,
+  Context,
+  APIGatewayProxyResult,
+  APIGatewayProxyCallback,
+} from 'aws-lambda';
+
+type LambdaHandler = (
+  event: APIGatewayProxyEvent,
```

Comment:
`event` isn't always `APIGatewayProxyEvent` (same for other variables). Probably an auto-generation error?

Reply:
Fixed